src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -488,18 +488,7 @@ public void testLoadPropertiesIoException() throws Exception {
         final Method method = Main.class.getDeclaredMethod("loadProperties", param);
         method.setAccessible(true);
         try {
-            if (System.getProperty("os.name").toLowerCase(Locale.ENGLISH).startsWith("windows")) {
-                // https://support.microsoft.com/en-us/kb/177506 but this only for NTFS
-                // WindowsServer 2012 use Resilient File System (ReFS), so any name is ok
-                final File file = new File(File.separator + ":invalid");
-                if (file.exists()) {
-                    file.delete();
-                }
-                method.invoke(null, new File(file.getAbsolutePath()));
-            }
-            else {
-                method.invoke(null, new File(File.separator + "\0:invalid"));
-            }
+            method.invoke(null, new File("."));
```

Comment:
Why did we remove all the code above?

Reply:
We do not need a special hack for windows if a folder name used instead of a file.
The exception will be thrown as expected.